changelog/12281.feature.rst

```diff
@@ -0,0 +1,8 @@
+Added support for keyword matching in marker expressions.
+
+Now tests can be selected by marker keyword arguments.
+Supported values are :class:`int`, (unescaped) :class:`str`, :class:`bool` & :data:`None`.
+
```

Comment:
It would be nice to add an example here, for people skimming the changelog without clicking through to the link.

Reply:
@lovetheguitar I think, it's nice to have examples in such places, but perhaps only small ones. I like Ran's idea. If you make a PR for that, we could discuss something more concrete there, with dedicated review and polishing.

Reply:
@bluetech I believe the previous comment of @lovetheguitar was addressed to you, eh?